Server/.env.example

```diff
@@ -0,0 +1,6 @@
+ACCOUNT_SID=ACxxxx
```

Comment:
Is the Server component a copy/paste of the Node.js quickstart project? I wonder if there is a better strategy for this. Maybe a github substree so that we keep things single sourced.

Reply:
@aaalaniz could we use a subtree here?

Reply:
@idelgado the `Server` folder is a modified copy of the [node.js server quickstart](https://github.com/twilio/voice-quickstart-server-node). The structure is specialized so that it be deployed by the Twilio CLI Serverless plugin.

Reply:
Gotcha. I think the question is whether we could make those structural changes in that repository allowing it to be used standalone or with this project

Reply:
Technically they work slightly differently. The servlet endpoints used by the Serverless plugin need to follow the format of Twilio Functions and let Functions handle the API endpoint redirect:

```
exports.handler = function(context, event, callback) {
    ...
    callback();
}
```

whereas the original node.js server quickstart is basically a full functioning web service which takes care of everything.

Reply:
Right. Can you write the code in such a way that you can reuse the functions and use node.js or functions?